src/Microsoft.ML.Data/DataLoadSave/EstimatorChain.cs

```diff
@@ -71,7 +71,12 @@ public SchemaShape GetOutputSchema(SchemaShape inputSchema)
         public EstimatorChain<TNewTrans> Append<TNewTrans>(IEstimator<TNewTrans> estimator, TransformerScope scope = TransformerScope.Everything)
             where TNewTrans : class, ITransformer
         {
-            Contracts.CheckValue(estimator, nameof(estimator));
+            if(estimator==null)
```

Comment:
>if(estimator==null) [](start = 12, length = 19)

if you use Visual Studio, press `Ctrl-k-d` to fix the spacing. Otherwise, please fix the spacing :)